miqa/core/tests/test_rest.py

```diff
@@ -62,6 +62,33 @@ def test_projects_list(user_api_client, project, user):
         }
 
 
+@pytest.mark.django_db
+def test_project_status(
+    project,
+    experiment,
+    scan_factory,
+    scan_decision_factory,
+    user_factory,
+):
+    # 3 of 5 scans marked as complete
+    decisions = [
+        ('U', 'tier_1_reviewer'),
+        ('U', 'tier_2_reviewer'),
+        ('UN', 'tier_1_reviewer'),  # needs tier 2 review
+        ('UN', 'tier_2_reviewer'),
+        ('Q?', 'tier_1_reviewer'),  # needs tier 2 review
+    ]
+    scans = [scan_factory(experiment=experiment) for i in range(len(decisions))]
+    for decision, scan in zip(decisions, scans):
+        decider = user_factory()
+        assign_perm(decision[1], decider, project)
+        scan_decision_factory(scan=scan, creator=decider, decision=decision[0])
+    project.refresh_from_db()
```

Comment:
```suggestion
```

I think this line might not be necessary

Reply:
You're right. I had figured the creation of subsequent objects may need a refresh, but it works without it, so I suppose that's only necessary for changes directly to the object. Changed in [ed33350](https://github.com/OpenImaging/miqa/pull/567/commits/ed33350fc09eb7a9b0d3c12d0434e45943cfb627)